Core/clim-core/panes.lisp

```diff
@@ -2314,6 +2314,12 @@ SCROLLER-PANE appear on the ergonomic left hand side, or leave set to
    :text-style (make-text-style :sans-serif nil nil))
   (:documentation ""))
 
+(defmethod (setf clime:label-pane-label) :after (new-value (pane label-pane))
+  (let ((region (sheet-region pane)))
+    (multiple-value-bind (x1 y1 x2 y2) (rectangle-edges* region)
+      (draw-rectangle* pane x1 y1 x2 y2 :ink (pane-background pane)))
+    (repaint-sheet pane region)))
```

Comment:
I would prefer just `(repaint-sheet pane (sheet-region pane))` here, but adding `(draw-rectangle* pane x1 (- ty a) x2 (+ ty d) :ink (pane-background pane))` before the `draw-text*` call in `handle-repaint`. That way, the label will always be drawn onto a freshly cleared background, no matter what caused the repaint. E.g. repainting the pane without changing the label would no longer lead to the label text appearing almost bold caused by overdrawing the anti-aliased glyph. This effect is visible in the "A" part of the label in the following screenshot:

![screenshot](https://user-images.githubusercontent.com/150189/66703666-4341e980-ed15-11e9-8eba-c1caa94ead05.png)

Reply:
Done. This is indeed better solution.